third_party/config/build/release.yaml

```diff
@@ -386,6 +386,9 @@ spec:
         volumeMounts:
         - mountPath: /etc/config-logging
           name: config-logging
+        resources:
```

Comment:
This file gets integrated from the build repo and changes here will be overridden next time we integrate. I would rather have us remove these changes. If these are needed, it should be done in the build repo.

Reply:
doi -meant to get these when I remove the istio ones. Should be gone now